contrib/packs/actions/install.meta.yaml

```diff
@@ -21,5 +21,5 @@
       default: "master"
     register:
       type: "string"
-      default: "actions"
+      default: "actions,aliases,sensors"
```

Comment:
I've tried with arrays first, but it's converted into string. See:
https://github.com/StackStorm/st2/issues/1231#issuecomment-111285096